raiden/transfer/channel.py

```diff
@@ -1435,6 +1478,37 @@ def handle_receive_directtransfer(
     return TransitionResult(channel_state, events)
 
 
+def handle_refundtransfer(
+        channel_state: NettingChannelState,
+        refund: ReceiveTransferRefund,
+):
+    # TODO model it from handle_receive_lockedtransfer and use the is_valid_refund()
```

Comment:
this todo is done, right?

Reply:
correct  :)

Reply:
But I would still like a check of the `is_valid_refund()`, could be missing some checks